linera-rpc/src/grpc/conversions.rs

```diff
@@ -1207,21 +1209,20 @@ pub mod tests {
             Round::SingleLeader(2),
             vec![(
                 ValidatorName::from(key_pair.public()),
-                Signature::new(&Foo("signed".into()), &key_pair),
+                AuthoritySignature::new(&Foo("signed".into()), &key_pair),
             )],
         )
         .lite_certificate()
         .cloned();
-        let public_key = SigningKey::generate().public();
+        let key_pair = AccountPrivateKey::generate();
         let block_proposal = BlockProposal {
             content: ProposalContent {
                 block: get_block(),
                 round: Round::SingleLeader(4),
                 outcome: Some(outcome),
             },
-            owner: Owner::from(public_key),
-            public_key,
-            signature: Signature::new(&Foo("test".into()), &SigningKey::generate()),
+            public_key: key_pair.public(),
+            signature: AccountSignature::new(&Foo("test".into()), &key_pair),
```

Comment:
A block proposal should be signed by an account key.

Reply:
And that is what's happening here, isn't it? I use `&key_pair` which is generated as `AccountPrivateKey::generate()`.

Reply:
Ah, right. :facepalm: